zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -85,35 +80,30 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.rms_current_ph_c.name,
             config=REPORT_CONFIG_OP,
         ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_b.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_c.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.rms_voltage.name,
             config=REPORT_CONFIG_OP,
         ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.rms_voltage_max.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.ac_frequency.name,
             config=REPORT_CONFIG_OP,
         ),
-        AttrReportConfig(
-            attr=ElectricalMeasurement.AttributeDefs.ac_frequency_max.name,
-            config=REPORT_CONFIG_DEFAULT,
-        ),
     )
+    ZCL_POLLING_ATTRS = {
```

Comment:
Actually, we might want `ZCL_POLLING_ATTRS` to be a `list` instead of a `set`, so that the attributes are always polled in the same (expected) order. So, just `ZCL_POLLING_ATTRS = [...]`.

Since we don't poll the divider/multiplier here at the moment, this might not matter, but I think it would be better anyways and keep the same order from before when a (sorted) dict was used.